<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Visualizer</class>
 <widget class="QWidget" name="Visualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualizer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 189, 182);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="ControlPanelGroup">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 75, 75);

</string>
     </property>
     <layout class="QHBoxLayout" name="PanelPlaceLayout">
      <item>
       <spacer name="PanelSpacer_Left">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="SortTypeSetLayout">
        <item>
         <widget class="QLabel" name="sortTypeSetTitle">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>Sort Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sortType">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <item>
           <property name="text">
            <string>Bubble Sort</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Insertion Sort</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Selection Sort</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Merge Sort</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Quick Sort</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="PanelSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DelaySetLayout">
        <item>
         <widget class="QLabel" name="delaySetLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>Delay:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="delay">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delaySet">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#delaySet{
	background-color: rgb(8, 0, 108);
	color: rgb(238, 238, 236);
}
#delaySet:hover {	
	color: rgb(237, 212, 0);
}
</string>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="PanelSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="BarsSetLayout">
        <item>
         <widget class="QLabel" name="barsSetTitle">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>Bars:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bars">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(238, 238, 236);</string>
          </property>
          <property name="minimum">
           <number>20</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="barsSet">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#barsSet{
	background-color: rgb(8, 0, 108);
	color: rgb(238, 238, 236);
}
#barsSet:hover {	
	color: rgb(237, 212, 0);
}
</string>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="PanelSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="OrderSetLayout">
        <item>
         <widget class="QPushButton" name="randomize">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#randomize{
	
	background-color: rgb(206, 92, 0);
}
#randomize:hover{
	
	background-color: rgb(255, 82, 0);
}</string>
          </property>
          <property name="text">
           <string>Randomize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="descending">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#descending{
	
	background-color: rgb(206, 92, 0);
}
#descending:hover{
	
	background-color: rgb(255, 82, 0);
}</string>
          </property>
          <property name="text">
           <string>Descending</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="PanelSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="ExecLayout">
        <item>
         <widget class="QPushButton" name="run">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(238, 238, 236);</string>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="PanelSpacer_Right">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="WorkAreaSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1259</width>
       <height>632</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="StatusBarGroup">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(75, 75, 75);</string>
     </property>
     <layout class="QHBoxLayout" name="StatusBarLayout">
      <item>
       <layout class="QHBoxLayout" name="StatusLayout">
        <item>
         <widget class="QLabel" name="stausTitle">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(156, 156, 156);</string>
          </property>
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="status">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>Not Running</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="StatusBarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>978</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="DelayDisplayLayout">
        <item>
         <widget class="QLabel" name="delayTitle">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(156, 156, 156);</string>
          </property>
          <property name="text">
           <string>Delay:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="delayLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="delayUnitLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(211, 215, 207);</string>
          </property>
          <property name="text">
           <string>µs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
